### PR TITLE
Feature/help section

### PR DESCRIPTION
**Pull request by greeshmareddyg**
Created "Help” section w/ith ”Support” and “FAQ” pages

https://github.ibm.com/brand/design-language/issues/25

### CODE REVIEW

File: src/pages/help/faq.mdx [hunk 1]
```diff
@@ -0,0 +1,161 @@
+---
+title: FAQ
+keywords: 'IBM,brand,faq'
```
Comment (alisonjoseph) [inline src/pages/help/faq.mdx]: We should probably add a `description:` field here also, thats what shows up when someone uses the search.

**Comment by alisonjoseph** [comment]
Make sure to request reviewers to ensure someone is looking at your PRs and they don't get lost. Need at least one design/one dev review.

File: src/pages/help/faq.mdx (hunk 1 above)
Comment (greeshmareddyg) [reply src/pages/help/faq.mdx]: added description..

File: src/pages/help/faq.mdx [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+- [Everything else](https://github.com/carbon-design-system/issue-tracking/issues/new)
+
+</AccordionItem>
+
+</Accordion>
+</AccordionGroup>
+
+## I still have a question
+
+If your questions has not been answered after thoroughly browsing this section, as well as the entire IBM Design Language site, [here]()'s how to reach us directly.
```
Comment (jnm2377) [inline src/pages/help/faq.mdx]: this link is missing a URL

File: src/pages/help/support.mdx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+## Questions
+
+Please take some time to explore the content on this website before reaching out to the IBM Design team. The site is very comprehensive and most of the guidelines and components are well documented. However, if you still have a specific question:
+
+
+<ol class="bx--list--ordered">
+<li class="bx--list__item">
+
+Please check whether there is an existing answer on our [FAQ page]()
```
Comment (jnm2377) [inline src/pages/help/support.mdx]: this link is also missing the URL

File: src/pages/help/support.mdx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+
+</li>
+<li class="bx--list__item">
```
Comment (jnm2377) [inline src/pages/help/support.mdx]: You also don't have to use html for this, you can just use tick marks or numbers for lists in MDX:

i.e. 
```md
- this is an unordered list item
- etc
- etc

1. this is an ordered list item
2. etc
3. etc
```

File: src/pages/help/support.mdx [hunk 5]
```diff
@@ -0,0 +1,111 @@
...
+ [Digital Design FAQ](https://www.ibm.com/standards/web/faq.html)
+
+</li>
+<li class="bx--list__item">
+
+ [Brand Center FAQ](https://www.ibm.com/brand/help/faq)
+
+</li>
+
+</ol>
```
Comment (jnm2377) [inline src/pages/help/support.mdx]: same as above

File: src/pages/help/support.mdx [hunk 6]
```diff
@@ -0,0 +1,111 @@
...
+
+</li>
+</ol>
+</li>
+<li class="bx--list__item">
+
+If you do not have access to IBM’s Slack channels, please email your question to [designlanguage@us.ibm.com]() Messages will be addressed by the IBM Brand Experience Design team as quickly as possible.
+
+</li>
+</ol>
```
Comment (jnm2377) [inline src/pages/help/support.mdx]: same as above

File: src/pages/help/support.mdx [hunk 7]
```diff
@@ -0,0 +1,111 @@
...
+
+
+- Start here with [IBM Design Essentials Course](https://w3.ibm.com/design/essentials). These six sessions will cover the must-know essentials, so you can understand, design, and implement with the new IBM Design Ethos, Language, and System. <br/>
+- Explore the [Gallery](https://www.ibm.com/design/language/gallery) of the new IBM Design Language in use.<br/>
+- Access elements and setup the [IBM Design kit](https://www.carbondesignsystem.com/resources/?&_ga=2.192292330.1162213957.1569426322-42838645.1566382348#theme-libraries) <br/>
+- Become a member of the [#ibm-design-kit](https://ibm-studios.slack.com/archives/CDT9TSUAJ) slack channel <br/>
+- Become a member of the [#ibm-design-language](https://ibm-studios.slack.com/messages/C0U35LB2B) channel <br/>
+- Request to become an IBM Design Language sponsor user group by emailing: [designlanguage@us.ibm.com]() with the subject: IDL Sponsor User Group request. Once you are part of the Sponsors User Group, participate in the IBM Design Language [Wake board](https://ibm.wake.com/space/20) <br/>
```
Comment (jnm2377) [inline src/pages/help/support.mdx]: link is missing URL

File: src/pages/help/support.mdx [hunk 8]
```diff
@@ -0,0 +1,111 @@
...
+
+More enablement materials will be developedwill be available as soon as possible. Please check back here again soon.
```
Comment (jnm2377) [inline src/pages/help/support.mdx]: this sentence has some errors...

File: src/pages/help/support.mdx (hunk 4 above)
Comment (greeshmareddyg) [reply src/pages/help/support.mdx]: I have used that before but they wanted the carbon list component.

File: src/pages/help/support.mdx (hunk 8 above)
Comment (greeshmareddyg) [reply src/pages/help/support.mdx]: fixed it.

File: src/pages/help/support.mdx (hunk 7 above)
Comment (greeshmareddyg) [reply src/pages/help/support.mdx]: It's just the email hyperlink no URL for that.

File: src/pages/help/faq.mdx (hunk 2 above)
Comment (greeshmareddyg) [reply src/pages/help/faq.mdx]: added slack channel.

File: src/pages/help/support.mdx (hunk 3 above)
Comment (greeshmareddyg) [reply src/pages/help/support.mdx]: that is TBD.

**Comment by greeshmareddyg** [comment]
@alisonjoseph @jnm2377 @sadek could you please review the PR.

**Comment by alisonjoseph** [comment]
IS this ready to merge @greeshmareddyg?

**Comment by greeshmareddyg** [comment]
@alisonjoseph we are expecting more changes for this section also..

**Comment by alisonjoseph** [comment]
@greeshmareddyg ok sounds good. I just updated the support.mdx page to fix the markdown so the lists should disply correctly. So just make sure you pull latest before updating 😄

**Comment by greeshmareddyg** [comment]
Sure, Thank you @alisonjoseph

File: src/pages/help/support.mdx (hunk 4 above)
Comment (vpicone) [reply src/pages/help/support.mdx]: @greeshmareddyg the theme supports nested lists

**Comment by vpicone** [comment]
@greeshmareddyg fixed your merge conflict, what's the status on this PR?

**Comment by vpicone** [comment]
Talked with @greeshmareddyg closing for now, can reopen when ready.

**Pull request closed without merging**